docs/en/observability/slo-troubleshoot.asciidoc

```diff
@@ -0,0 +1,255 @@
+[[slo-troubleshoot-slos]]
+= Troubleshoot service-level objectives (SLOs)
+
+++++
+<titleabbrev>Troubleshoot SLOs</titleabbrev>
+++++
+
+include::slo-overview.asciidoc[tag=slo-license]
+
+[WARNING]
+====
+Do Not edit, delete, or tamper any "internal" asset mentioned in this document, such as the transforms or ingest pipelines created by the SLO application.
+
+Do Not attempt to edit the `.slo-observability.*` indices mentioned in this document by overriding index templates or editing the settings/mappings.
+
+This document describes some implementation details that are subject to change.
+====
+
+This document provides an overview of common issues encountered when working with service-level objectives (SLOs). It explores the relationships between SLOs and other core functionalities within the stack, such as {ref}/transforms.html[transforms] and {ref}/ingest.html[ingest pipelines], highlighting how these integrations can impact the functionality of SLOs.
```

Comment:
@dedemorton : both sentences start with the same `this document provides`. I'd like your help here if possible ;)